src/pages/dataTableUtilities/getPageInfoColumns.js

```diff
@@ -86,8 +86,11 @@ const PER_PAGE_INFO_COLUMNS = {
     ['breachDuration', 'location'],
     ['numberOfAffectedBatches', 'affectedQuantity'],
   ],
+
   customerRequisitionItemDetail: [
     [
+      'numberOfDaysInAMonthString',
+      'numberOfDaysInAMonthFormula',
```

Comment:
It was confusing refering to the number of days in a month without specifying what that was